entries/m/my.invisalign.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "My Invisalign": {
+    "domain": "my.invisalign.com",
+    "tfa": [
+        "email"
```

Comment:
```suggestion
      "email"
```

Can you provide evidence of 2FA?

Reply:
![evidence](https://user-images.githubusercontent.com/68301104/225709693-ab71396d-5d3a-400c-8685-6769e50e760f.png)

Evidence of it asking you to check your mail.